assets/sass/v2/_okta-verify.scss

```diff
@@ -34,6 +34,10 @@
   .sameDevice-setup {
     text-align: center;
   }
+  
+  .sameDevice-setup.ov-enrolment-enabled {
```

Comment:
nit: typo?

```suggestion
  .sameDevice-setup.ov-enrollment-enabled {
```

Reply:
the vscode showed me `enrollment ` as typo .... will change to `enrollment ` to keep consistency with other places